CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to the "vscode-didact" extension will be documented in this
 
 ## 0.1.9
 
- - TBD
+ - Adding some basic autocompletion for Didact URIs in Markdown documents
```

Comment:
Optional: I'm sure we can find something that will make me dreaming a lot more than "basic completion" :-)

what about "Adding completion to ease creation of Didact URIs in markdown documents"? or describing more precisely what is available?